### PR TITLE
dev: Use eth-optimism/core-utils where applicable

### PR DESCRIPTION
**Pull request by kfichter**
## Description
Removes a bunch of duplicated code.

## Contributing Agreement

- [x] I have read and understood the [Optimism Contributing Guide and Code of Conduct](https://github.com/ethereum-optimism/optimism-monorepo/blob/master/.github/CONTRIBUTING.md) and am following those guidelines in this pull request.

### CODE REVIEW

File: test/contracts/OVM/accounts/OVM_ProxyEOA.spec.ts [hunk 1]
```diff
@@ -4,9 +4,10 @@ import { expect } from '../../../setup'
 import { ethers, waffle } from 'hardhat'
 import { ContractFactory, Contract, Wallet } from 'ethers'
 import { MockContract, smockit } from '@eth-optimism/smock'
+import { remove0x } from '@eth-optimism/core-utils'
 
 /* Internal Imports */
-import { remove0x } from '../../../helpers'
+import {} from '../../../helpers'
```
Comment (kfichter) [inline test/contracts/OVM/accounts/OVM_ProxyEOA.spec.ts:10]:
```suggestion
```

File: src/contract-defs.ts [hunk 2]
```diff
@@ -6,7 +6,7 @@ import { Interface } from 'ethers/lib/utils'
 export const getContractDefinition = (name: string, ovm?: boolean): any => {
   const match = glob.sync(
     path.resolve(__dirname, `../artifacts`) +
-      `/**/${name}${ovm ? '.ovm' : ''}.json`
+      `/**/${name}${ovm ? '-ovm' : ''}.json`
```
Comment (tynes) [inline src/contract-defs.ts:9]: Any test coverage around this?

Comment (kfichter) [reply src/contract-defs.ts:9]: Nope :-( but this is an effect of the latest plugins update.

**Review by maurelian — APPROVED** [review]
This all LGTM, and build/tests are passing.

**Review by maurelian — APPROVED** [review]
Fix looks good. Merge if checks pass.

**Pull request merged**